src/apps/dashboard/routes/playback/resume.tsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import Page from 'components/Page';
+import globalize from 'lib/globalize';
+import Alert from '@mui/material/Alert';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Stack from '@mui/material/Stack';
+import TextField from '@mui/material/TextField';
+import Typography from '@mui/material/Typography';
+import { type ActionFunctionArgs, Form, useActionData, useNavigation } from 'react-router-dom';
+import { ActionData } from 'types/actionData';
+import { useConfiguration } from 'hooks/useConfiguration';
+import Loading from 'components/loading/LoadingComponent';
+import ServerConnections from 'components/ServerConnections';
+import { getConfigurationApi } from '@jellyfin/sdk/lib/utils/api/configuration-api';
+
+export const action = async ({ request }: ActionFunctionArgs) => {
+    const api = ServerConnections.getCurrentApi();
+    if (!api) throw new Error('No Api instance available');
+
+    const { data: config } = await getConfigurationApi(api).getConfiguration();
+    const formData = await request.formData();
+
+    const minResumePercentage = formData.get('MinResumePercentage')?.toString();
+    const maxResumePercentage = formData.get('MaxResumePercentage')?.toString();
+    const minAudiobookResume = formData.get('MinAudiobookResume')?.toString();
+    const maxAudiobookResume = formData.get('MaxAudiobookResume')?.toString();
+    const minResumeDuration = formData.get('MinResumeDuration')?.toString();
+
+    if (minResumePercentage) config.MinResumePct = parseInt(minResumePercentage, 10);
+    if (maxResumePercentage) config.MaxResumePct = parseInt(maxResumePercentage, 10);
+    if (minAudiobookResume) config.MinAudiobookResume = parseInt(minAudiobookResume, 10);
+    if (maxAudiobookResume) config.MaxAudiobookResume = parseInt(maxAudiobookResume, 10);
+    if (minResumeDuration) config.MinResumeDurationSeconds = parseInt(minResumeDuration, 10);
+
+    await getConfigurationApi(api)
+        .updateConfiguration({ serverConfiguration: config });
+
+    return {
+        isSaved: true
+    };
+};
+
+const Resume = () => {
```

Comment:
We should prefer exporting a named `Component` since that is the react router way... using the default export as the component is something I added for backwards compatibility with our existing pages

```suggestion
export const Component = () => {
```

then set the display name `Component.displayName = 'ResumePage';`